vignettes/secuTrialR-package-vignette.Rmd

```diff
@@ -474,14 +474,28 @@ plot_recruitment(ctu05_data_bern)
 ... or Bern and Berlin.
 \vspace{5pt}
 
-```{r, fig.height = 3.9, fig.width = 8}
+```{r, fig.height = 3.8, fig.width = 8}
 # keep only Bern and Berlin
-ctu05_data_bern_berlin <- subset_secuTrial(ctu05_data, centre = c("Inselspital Bern (RPACK)",
-                                                                  "Charité Berlin (RPACK)"))
+ctu05_data_bern_berlin <- subset_secuTrial(ctu05_data,
+                                           centre = c("Inselspital Bern (RPACK)",
+                                                      "Charité Berlin (RPACK)"))
 # plot
 plot_recruitment(ctu05_data_bern_berlin)
 ```
 
+# `as.data.frame`
+
+This vignette has been working with the `secuTrialdata` object as a `list`.
+For some users, working with a `list` can be tiresome so `secuTrialR` provides an
+`as.data.frame()` method to save the `data.frame`s in the `list` to
+an environment of your choice.
+\vspace{5pt}
+
+```{r}
+# add files to global environment
+as.data.frame(ctu05_data)
+```
+
```

Comment:
I would then show that the individual dataframes are in the global environment by showing `ls()`